css/index.css

```diff
@@ -0,0 +1,157 @@
+html{
+    font-family: 'Playfair Display', serif;
+}
+
+body{
+    color: #262626;
+    background-color: 242, 242, 242;
```

Comment:
This is not a valid color format, you either need to specify `rgb` or use hex format like you did on `color`:
`rgb(242, 242, 242)` or `#f2f2f2`